modules/cockpit/manifests/init.pp

```diff
@@ -0,0 +1,22 @@
+class cockpit {
+	if $::osfamily == 'RedHat' {
+		package { 'cockpit' :
+			ensure => installed,
+		}
+		service {'cockpit' :
```

Comment:
this is great, awesome job
literally the only thing i would change is moving this 'service' below the 'file'
reasoning being: in Puppet 4 and newer, Puppet automatically executes resources in the order they're written in the manifest.

what you have written is technically correct because the order is explicitly defined by you with your 'require' and 'notify' statements. those are awesome and the 'notify' will automatically restart the 'service' if the config file changes (badass).

well done.